commons/src/main/java/org/infinispan/commons/util/concurrent/ConcurrentHashSet.java

```diff
@@ -1,4 +1,4 @@
-package org.infinispan.util.concurrent;
+package org.infinispan.commons.util.concurrent;
```

Comment:
I've created a JIRA (mostly as a reminder) for moving it in ISPN9 ([ISPN-6297](https://issues.jboss.org/browse/ISPN-6297))